openpathsampling/experimental/simstore/serialization_helpers.py

```diff
@@ -8,7 +8,10 @@
 from .tools import flatten_all, nested_update, group_by_function
 from .tools import is_iterable, is_mappable, is_numpy_iterable
 from . import tools
-from .class_lookup import is_storage_iterable, is_storage_mappable
+from .class_lookup import (
+    is_storage_iterable, is_storage_mappable, is_storage_string,
+    has_storage_uuid,
+)
 from .proxy import GenericLazyLoader
 from .uuids import (
     has_uuid, get_uuid, set_uuid, encode_uuid, decode_uuid, encoded_uuid_re,
```

Comment:
shouldn't `has_uuid` be dropped from the imports? (only reference is in commented out code)
Bit surprised linting did not catch this

Reply:
Turns out this was trickier than it appeared. In an early version of the file, `has_uuid` was defined in `serialization_helpers.py`. Then I split it off into `uuids.py`. But I allowed the transitive import.

Fixed that in [8bf1f31](https://github.com/openpathsampling/openpathsampling/pull/1157/commits/8bf1f31493719f0d55537bdea987796cfa282f23).